queries/pod/highlights.scm

```diff
@@ -0,0 +1,67 @@
+; A highlight file for nvim-treesitter to use
+
+[(pod_command)
+ (command)
+ (cut_command)] @keyword
+
+(command_paragraph
+  (command) @keyword
+  (#match? @keyword "^=head")
```

Comment:
You should use `lua-match` for these.

Reply:
is there a particular reason? I'd just rather be able to have the same captures available for when i test using the CLI

Reply:
Performance.

Reply:
_significant_ performance. I am afraid we hve to insist on that. 

(Queries are editor specific; I understand wanting to minimize the divergence, but it's not possible in general to keep them completely agnostic.)